### ISSUE TITLE
travis.yml builds the master branch on pull requests instead of the branch under test

### ISSUE THREAD

**Issue by n2ygk** [tienfuc/gdcmdtools#58]
The travis.yml has:
```
pip install -r requirements.txt && pip install git+https://github.com/timchen86/gdcmdtools.git@master
```
which overwrites the just checked-out pull request branch with the master branch.

This should be changed (I believe) to:
```
pip install -r requirements.txt
```

**Comment by n2ygk** [comment]
This appears to do the right thing, but the later tests in test.py are looking for credentials that are not available to travis.

**Comment by tienfuc** [comment]
n2ygk,
Thanks, I have fixed this issue on 7310ecc60e53e89c5c3f24dddbd7c5d06da8916f
